components/CustomIconButton.jsx

```diff
@@ -0,0 +1,26 @@
+import { View, Text, TouchableOpacity } from 'react-native'
+import React from 'react'
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+
+const CustomIconButton = ({ iconName, iconSize, iconColor, handlePress, containerStyles, isLoading, libName}) => {
+  
+  const IconComponent = libName === 'MaterialIcons' ? MaterialIcons : MaterialCommunityIcons;
+
+  return (
+    // border-secondary border-solid border-2 
+    <View className = "min-h-[{iconSize+2}]">
```

Comment:
tailwind doesnt support dynamic styles, read about it. also syntax of using dynamic values not correct